Source/WebCore/platform/graphics/Font.cpp

```diff
@@ -684,6 +684,28 @@ TextStream& operator<<(TextStream& ts, const Font& font)
     ts << font.description();
     return ts;
 }
+
+WTF::TextStream& operator<<(WTF::TextStream& ts, const GlyphBuffer& glyphBuffer)
+{
+    ts << "glyphBuffer: " << &glyphBuffer;
+    auto initialAdvance = glyphBuffer.initialAdvance();
+    ts << ", initial advance: w:" <<  width(initialAdvance) << " h:" << height(initialAdvance);
```

Comment:
Feel like we should not be using single character naming even for logging output. Might as well just be width/height.